component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -1131,7 +1131,7 @@ private String importUser(UserImportResultEntity userImportResultEntity,
 
     User existingUser = organizationService.getUserHandler().findUserByName(userName, UserStatus.ANY);
     if (existingUser != null) {
-      userImportResultEntity.addErrorMessage(userName, "USERNAME:ALREADY_EXISTS");
+      organizationService.getUserHandler().saveUser(user, true);
```

Comment:
Why do we save user ? to update fields ?

Reply:
yes